.gitignore

```diff
@@ -0,0 +1,4 @@
+node_modules
+bower_components
+*.swp
```

Comment:
*.swp and .DS_Store you can add to the .gitignore in your home directory. They aren't byproducts of the tools used by this repo but rather by tools used by your editing tools and OS.